examples/get-file-contents.js

```diff
@@ -0,0 +1,122 @@
+import {
+    PrivateKey,
+    FileContentsQuery,
+    Hbar,
+    FileCreateTransaction,
+    FileDeleteTransaction,
+    Client,
+    AccountId,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// Removed pino logger initialization
+const logger = console;
```

Comment:
Could you please refer to the Java example to view the Hedera logger I mentioned? Additionally, please make the necessary corrections in the code.

Reply:
have used the appropriate logger for javascript

Reply:
should i add SDK_LOG_LEVEL on the env file since its not available??? @ivaylogarnev-limechain

Reply:
Apologize for the confusion. I wasn't referring to the logger from the JavaScript. I was actually talking about the Hedera logger component, which you can see how it is used  and set up in the JavaScript examples. This logger is set up to log every interaction with the blockchain in the console.

hint:
```
import {
    Logger,
    LogLevel,
} from "@hashgraph/sdk";
```

Reply:
resolved

Reply:
@ivaylogarnev-limechain completed